components/compliance-service/reporting/util/csv.go

```diff
@@ -82,3 +83,11 @@ func ReportToCSV(report *reportingapi.Report) (string, error) {
 	}
 	return content, nil
 }
+
+func maxCharLimit(item string) string {
+	if utf8.RuneCountInString(item) > 32000 {
+		countToReturn := 32000 - 52
+		return fmt.Sprintf("character limit exceeds 32,000. truncating message: %s", string([]rune(item)[0:countToReturn]))
```

Comment:
nitpick: Instead of hardcoding the length of the message, you could have the message in a variable and take its length instead. This would be useful if we ever wanted to tweak the message, etc.